tests/drivers/charger/sbs_charger/boards/emulated_board.conf

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023 Cirrus Logic, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_EMUL=y
+CONFIG_EMUL_SBS_CHARGER=y
+CONFIG_I2C=y
```

Comment:
looks redundant, sbs driver should select i2c

Reply:
Ack